### PR TITLE
Enable more xo rules

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: js/src/offcanvas.js [hunk 1]
```diff
@@ -257,7 +257,10 @@ EventHandler.on(document, EVENT_CLICK_DATA_API, SELECTOR_DATA_TOGGLE, function (
 })
 
 EventHandler.on(window, EVENT_LOAD_DATA_API, () =>
-  SelectorEngine.find(OPEN_SELECTOR).forEach(el => Offcanvas.getOrCreateInstance(el).show())
+  // eslint-disable-next-line unicorn/no-array-for-each
+  SelectorEngine.find(OPEN_SELECTOR).forEach(el => {
```
Comment (XhmikosR) [inline js/src/offcanvas.js:261]: This we might be able to change to `for...of`

Comment (GeoSot) [reply js/src/offcanvas.js:261]: Do I miss any difference compared to other changed `SelectorEngine.find`?

Comment (XhmikosR) [reply js/src/offcanvas.js]: It wasn't autofixable due to the missing brackets.

**Comment by GeoSot** [comment]
Seems good to my eyes, although I would prefer to address it, on our next minor version

**Review by GeoSot — APPROVED** [review]

**Pull request merged**